src/isa/riscv64/instr/rvf/decode.h

```diff
@@ -67,38 +67,46 @@ static inline def_DHelper(r2fr){
 #ifdef CONFIG_RVV
 bool vp_enable();
 def_THelper(vload) {
-  def_INSTR_TAB("??? ?00 ? ????? ????? ??? ????? ????? ??", vlduu);
-  def_INSTR_TAB("??? ?01 ? ????? ????? ??? ????? ????? ??", vldxu);
-  def_INSTR_TAB("??? ?10 ? ????? ????? ??? ????? ????? ??", vldsu);
-  def_INSTR_TAB("??? ?11 ? ????? ????? ??? ????? ????? ??", vldxu);
+  def_INSTR_TAB("??? 000 ? ?0000 ????? ??? ????? ????? ??", vle);
+  def_INSTR_TAB("??? 000 ? 01000 ????? ??? ????? ????? ??", vlr);
+  def_INSTR_TAB("??? 000 ? 01011 ????? ??? ????? ????? ??", vlm);
+  def_INSTR_TAB("??? 001 ? ????? ????? ??? ????? ????? ??", vlxe);
+  def_INSTR_TAB("??? 010 ? ????? ????? ??? ????? ????? ??", vlse);
+  def_INSTR_TAB("??? 011 ? ????? ????? ??? ????? ????? ??", vlxe);
```

Comment:
PLZ add some comments at the end of each line, like VLE, VLUXE, VLSE, VLOXEI, VLM